core/vm/instructions.go

```diff
@@ -372,7 +372,7 @@ func opReturnDataCopy(pc *uint64, interpreter *EVMInterpreter, scope *ScopeConte
 
 func opExtCodeSize(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
 	slot := scope.Stack.Peek()
-	slot.SetUint64(uint64(interpreter.evm.IntraBlockState().GetCodeSize(slot.Bytes20())))
+	slot.SetUint64(uint64(len(interpreter.evm.IntraBlockState().ResolveCode(slot.Bytes20()))))
```

Comment:
Please remove `IntraBlockState.GetCodeSize` because it's unused now.

Reply:
I instead added a ResolveCodeSize, since that would avoid loading code in memory (to find the length) when only the size but not the code is needed. Like in case of extcodesize.